app/src/main/java/com/fmsys/snapdrop/MainActivity.java

```diff
@@ -191,6 +192,14 @@ public boolean onShowFileChooser(WebView mWebView, ValueCallback<Uri[]> filePath
             return true;
         }
 
+        @Override
+        public boolean onCreateWindow(WebView view, boolean dialog, boolean userGesture, android.os.Message resultMsg) {
```

Comment:
Codacy found an issue: [Parameter view should be final.](https://app.codacy.com/gh/fm-sys/snapdrop-android/pullRequest?prid=6298520)